src/docs/patterns/empty-states.mdx

```diff
@@ -1,64 +1,89 @@
+Empty states should include a large icon or graphic followed a brief headline/message. If necessary, additional explanatory text may follow.
 
--   [@netbasal/ngx-content-loader](https://www.npmjs.com/package/@netbasal/ngx-content-loader) (Angular, Ionic)
--   [ion-skeleton-text](https://ionicframework.com/docs/api/skeleton-text) (Ionic)
--   [placeholder-loading](https://www.npmjs.com/package/placeholder-loading) (React, Angular, Ionic)
--   [react-content-loader](https://www.npmjs.com/package/react-content-loader) (React)
--   [rn-placeholder](https://www.npmjs.com/package/rn-placeholder) (React Native)
+The text on the screen should clearly indicate why there is no data to display, and what (if anything) can be done to make data available. The text should be minimal, but enough to convey the necessary message.
 
-### Refreshing Data
+If the data is empty but could be populated by a user action (such as adding a device to an empty device list), there should be a call-to-action button that will allow the user to add data.
 
-When refreshing data, there are several approaches you can take. If you are showing real-time data (either via polling or a push mechanism from the server), there is no need to show large loading indicators. You may wish to show a small indicator or at least mention when the data was last refreshed so users can identify if the data is stale.
+<ImageGrid images={[Image1, Image2]} />
 
-If your users can manually refresh, you may wish to show the placeholder elements (as with initial load), or simply refresh the data behind the scenes (as with real-time). Ultimately, this should depend on how long it takes to refresh the data.
+## Variations
 
-## Empty States
+### Full-screen
 
-<ImageGrid images={[Image3, Image4]} />
-<MaterialDesignDescription title={'Empty States'} url={'https://material.io/design/communication/empty-states.html'} />
+In the most common cases, Empty States should be used full-screen as the only content on a page. In this situation, the indicator should be centered on the page. In some cases, you may wish to use a large graphic as the background of the entire page as well (such as for features that will be coming soon).
 
-<!-- @TODO switch empty-state links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+### Element-level
 
-<DemoCard
-    repository={'empty-states'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-empty-state--with-full-config'}
-    react={'https://pxblue-components.github.io/react/?path=/story/components-empty-state--with-full-config'}
-    ionic
-    reactNative
-/>
+While less common, empty states can also be used as an indicator for individual sections of a page rather than the entire page (such as a single card having no data on a dashboard / overview screen). In these cases, it may be appropriate to use a smaller icon and text depending on the available space.
```

Comment:
```suggestion
While less common, empty states can also be used as an indicator for individual sections of a page rather than the entire page (such as a single card having no data on a dashboard/overview screen). In these cases, it may be appropriate to use a smaller icon and text depending on the available space.
```

Reply:
We're inconsistent with this. It's about 50/50 using the extra spaces versus not. I kind of prefer it for readability, though grammatically, you are correct.

Reply:
I also personally prefer the spaced style -- I'd say pick one side and stick with it.